torch/csrc/jit/codegen/cuda/scheduler/utils.h

```diff
@@ -51,7 +51,24 @@ size_t mergeNonReduction(
 
 TORCH_CUDA_CU_API void parallelizeAllLike(
     TensorView* reference_tv,
-    const std::vector<TensorView*>& all_tvs);
+    int64_t pos = -1,
```

Comment:
Please add  a comment

Reply:
Added on top of `parallelizeAllLike`. And you are right, `pos` means selecting the first `pos` IDs.